### ISSUE TITLE
Add skip to content links

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#831]
From our accessibility audit:

> A skip to content link is appropriate all on pages, but on [the packages page], I think it becomes necessary because there is a lot of stuff to process before finding [the embedded .rst]. If you need an example of such links, go to https://github.com/ and press the tab key on your keyboard. A skip to content link should appear.

**Comment by umeshksingla** [comment]
Can I help with this issue? Tell me which page you are talking about (I think it is warehouse/warehouse/templates/packaging/detail.html) and where should I put the tabindex?

**Comment by nlhkabu** [comment]
Hi @umeshksingla - for now it would be great to include this link at the top right of the header.
You will be able to find this in base.html (https://github.com/pypa/warehouse/blob/master/warehouse/templates/base.html)

The link should *only* show once a user has hit the tab button on their keyboard.

**Comment by wlonk** [comment]
@umeshksingla: are you still working on this? I don't see a PR for it, and would be happy to knock it out for you if I'm not duplicating work.

**Comment by dstufft** [comment]
I'm going to drop this out of the Launch milestone, not because I think it's not important, but because current PyPI doesn't have this, so not having it won't be a regression and thus it can be done anytime before or after Launch.
